pages/[username].js

```diff
@@ -1,146 +1,155 @@
-import Head from "next/head";
-import ReactMarkdown from "react-markdown";
-import { abbreviateNumber } from "js-abbreviation-number";
+import Head from 'next/head';
```

Comment:
I think the coding style of the project is `"`

Reply:
Should be fixed now